settings.gradle.kts

```diff
@@ -18,7 +18,7 @@ dependencyResolutionManagement {
             alias("kord-common").to("dev.kord", "kord-common").versionRef("kord")
             alias("kord-rest").to("dev.kord", "kord-rest").versionRef("kord")
             alias("kord-gateway").to("dev.kord", "kord-gateway").versionRef("kord")
-            alias("ktor-server-netty").to("io.ktor:ktor-server-netty:1.6.7")
+            alias("ktor-server-netty").to("io.ktor:ktor-server-netty:2.0.0-beta-1")
```

Comment:
This should be updated to 2.0.0

Reply:
Fixed it, thanks :) https://github.com/LorittaBot/DiscordInteraKTions/commit/5fc5fbf9275ba28b338cfeb83d8d3ac62045aee0